build-configs/TypographyType.ts

```diff
@@ -1,7 +1,7 @@
 import * as CSS from 'csstype'
 import React from 'react'
 
-type TypographyVariant =
+export type TypographyVariant =
```

Comment:
🤔 Do we still use this somewhere?

Reply:
I used it for variant type:
<img width="362" height="300" alt="Screenshot 2025-11-25 at 12 17 43 PM" src="https://github.com/user-attachments/assets/fec6a399-9b76-4ef0-aa21-67e7437fb7dc" />